optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -339,6 +339,20 @@ def to(self, device: str):
     def device(self) -> str:
         return self._device.lower()
 
+    @property
+    def current_height(self) -> int:
```

Comment:
Maybe just call properties height and width but I would let @echarlaix to decide here.

Reply:
renamed